<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
 background-color: transparent;
 border-width: 1px;
 border-radius: 10px; 
}

QMenu
{
        color:rgb(0, 0, 0);
        background-color:#F5F5F5;
        border:none;
}
QMenu::item
{
        color:rgb(0, 0, 0);
        background-color:#F5F5F5;
}
QMenu::item:selected
{
        color:rgb(0,0,0);
        background-color:#FFFFFF;
}


 

/* 不可用(操作)选项颜色灰色*/

QMenu::item:!enabled   

{
color:rgb(128，136,145);

background-color:rgba(255,255,255,1);

}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/img/head2.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>6</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AvenirNext LT Pro Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>yychen</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/img/note.jfif);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AvenirNext LT Pro Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>16</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AvenirNext LT Pro Medium</family>
    </font>
   </property>
   <property name="text">
    <string>on/off</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
